pandas/io/pytables.py

```diff
@@ -2517,7 +2517,7 @@ def write_array(self, key, value, items=None):
         transposed = False
 
         if com.is_categorical_dtype(value):
-            raise NotImplementedError("cannot store a category dtype")
+            raise NotImplementedError('Cannot store a category dtype in a HDF5 dataset that uses format="fixed". Use format="table".')
```

Comment:
fine but this will fail PEP, break the line:

```
raise NotImplementedError(" ......... "
                                             " ......... "
                                             " ..........")
```